src/renderer/base/renderer.cpp

```diff
@@ -1036,7 +1037,10 @@ void Renderer::_PaintOverlay(IRenderEngine& engine,
         // Set it up in a Viewport helper structure and trim it the IME viewport to be within the full console viewport.
         Viewport viewConv = Viewport::FromInclusive(srCaView);
 
-        for (SMALL_RECT srDirty : engine.GetDirtyArea())
+        gsl::span<const til::rectangle> dirtyAreas;
+        LOG_IF_FAILED(engine.GetDirtyArea(dirtyAreas));

```

Comment:
(why'd we make it an HR return?)

Reply:
cuz it was the only one to not HR return on the interface, per our discussion, closing.